sites/fast-website/src/app/components/fast-frame/fast-frame.template.ts

```diff
@@ -70,6 +71,7 @@ export const FastFrameTemplate = html<FastFrame>`
                             </fast-radio-group>
                             <label for="accent-color-pickers">Accent color</label>
                             <fast-radio-group
+                            class="swatches"
```

Comment:
```suggestion
                                class="swatches"
```

Reply:
interesting - I don't recall formatting like this...and prettier should catch. I'm going to run it again.